src/vmm/src/default_syscalls/filters.rs

```diff
@@ -71,29 +92,32 @@ pub fn default_filter() -> Result<SeccompFilter, Error> {
                     libc::MADV_DONTNEED as u64
                 )?],],
             ),
-            allow_syscall(libc::SYS_mmap),
-            allow_syscall(libc::SYS_mremap),
```

Comment:
This was also triggered by musl for some customer workloads?

@aghecenco wasn't this triggered by our metrics system at some point (actually by the rust standard library code when musl when publishing metrics)

Reply:
Initially, I thought that it was only triggered by `libbacktrace` (which we dropped), since it was introduced by this [PR](https://github.com/firecracker-microvm/firecracker/commit/f62732f4020e76e39e1b311baa6c2b093249e97d)
@aghecenco pointed out that it was also used by this [scenario](https://github.com/firecracker-microvm/firecracker/issues/1336).

I will put it back in the allowlist, since it may needed by any re-allocation of a large buffer.